kitsune/questions/jinja2/questions/question_list.html

```diff
@@ -295,6 +304,22 @@ <h3 class="sidebar-subheading">{{ _('Filter by') }}</h3>
       {% for f, desc in filters.items() %}
       <li {{ filter|class_selected(f) }}><a href="{{ questions_url(filter=f, page=None) }}">{{ desc }}</a></li>
       {% endfor %}
+      <li class="sidebar-subheading sidebar-nav--heading-item">{{ _('Topic') }}</li>
```

Comment:
This seems to me like it should be moved under the `{% if topic_list %}` clause immediately below, because it looks odd on the page when there is no `topic_list`.

<img width="1414" alt="image" src="https://github.com/mozilla/kitsune/assets/3743693/c5f0f179-6a78-4338-ab5f-3e88cc42e4f8">